Language/Functions/Formal_Parameters/Optional_Formals/name_t02.dart

```diff
@@ -30,4 +30,7 @@ class C {
 main() {
   new A(_p: "Optional parameter names must not begin with an underscore!");
```

Comment:
Wouldn't this one be an error as well?

Reply:
No. Declaration of named parameters with the names `_` and `_p` is an error. But invocation of the function with these parameters seems a secondary error and reported for `_` only.

Reply:
Good catch!

I created https://github.com/dart-lang/language/issues/4062 in response to this behavior, I think we might as well make it an error.

But then we'll keep this as it is, and make changes if 4062 creates a need for that.